### PR TITLE
Add support for modeling activeItems in v-list

### LINKED ISSUE #456: Docs: Add "accordion" option to v-list
It would be nice for the docs to close other nested groups when opening a new one. This keeps the nav more manageable.

I assume this would be added as a new prop to the `v-list` component?

### LINKED ISSUE #462: Docs: Show full navigation when deep linking to a page
If you navigate to a specific page (eg: `/docs/guides/creating-a-project`), then we should make sure the navigation is automatically showing the correct sections/subsections expanded.

### PR DESCRIPTION
**Pull request by Nitwel**
closes #456 closes #462

### CODE REVIEW

**Comment by Nitwel** [comment]
This can now be reviewed.

File: app/src/modules/docs/components/navigation.vue [hunk 1]
```diff
@@ -23,22 +23,26 @@ function spreadPath(path: string) {
...
+		watch(() => props.path, (newPath) => {
+			_selection.value = spreadPath(newPath.replace('/docs',''))
+		})
 
 		const selection = computed({
 			get() {
 				return _selection.value
 			},
 			set(newSelection: string[]) {
+				console.log("newSelect: ", newSelection);
```
Comment (rijkvanzanten) [inline app/src/modules/docs/components/navigation.vue]: Found one!

Comment (rijkvanzanten) [inline app/src/modules/docs/components/navigation.vue]:
```suggestion
```

File: docs/nav-app.yaml [hunk 2]
```diff
@@ -12,6 +13,7 @@
 
 - name: Concepts
   icon: school
+  to: "/concepts/"
```
Comment (rijkvanzanten) [inline docs/nav-app.yaml:16]:
```suggestion
  to: "/concepts"
```

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**